app/lib/common/widgets/space_name_widget.dart

```diff
@@ -14,10 +19,12 @@ class SpaceNameWidget extends ConsumerWidget {
 
   Widget _buildSpaceName(BuildContext context, WidgetRef ref) {
     String spaceName =
-        ref.watch(roomDisplayNameProvider(spaceId!)).valueOrNull ?? '';
+        ref.watch(roomDisplayNameProvider(spaceId)).valueOrNull ?? '';
```

Comment:
Where do we intend to use this widget? Can you point me to the UI code ? Also we probably want to switch it to fallback ```spaceId``` rather than just showing empty ???

Reply:
Sure, will do!